inst/uvhydrograph/uvhydrograph.Rmd

```diff
@@ -2,13 +2,13 @@
 
 ## `r data[['title']]`  
 
-**Location:**  `r data[['reportMetadata']][['stationId']]` - `r data[['reportMetadata']][['stationName']]`
+**Location:**  `r getReportMetadata(data,stationId)` - `r getReportMetadata(data,stationName)`
```

Comment:
Tests are failing, I think you are supposed to pass in strings to the getReportMetadta function. EG: getReportMetadata(ts,'locationNumber',required=TRUE)